integration/330_application_edge_test.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Test short connections between processes"
+
+WEAVE_DOCKER_ARGS=$ADD_HOST_ARGS weave_on $HOST1 launch
+scope_on $HOST1 launch
+weave_on $HOST1 run -d --name nginx nginx
+weave_on $HOST1 run -dti --name client alpine /bin/sh -c "while true; do \
+	nc nginx.weave.local 80 || true; \
+	sleep 1; \
+done"
```

Comment:
Not supposed to be a short connection (they don't get applied between processes) - I'm testing procspy, which only looks at long connections.  Have updated comment and turned off conntrack for this test to make it more obvious.